### ISSUE TITLE
fish in 2012-2015 sequencing data without gen_ids

### ISSUE THREAD

**Issue by katcatalano** [pinskylab/genomics#6]
The R data file in this [file](https://github.com/katcatalano/parentage/blob/master/nogenid.rds) is a table of sample_ids that are in the sequencing data, but don't attach to a gen_id. Let me know if you want it in a different format.

**Comment by mstuart1** [comment]
I can't open that because it is private.  When I go to your GitHub page it says you have 0 repos.  Can you upload the file to this genomics repo?

**Comment by katcatalano** [comment]
Sorry about that, I uploaded the .rds to pinskylab/genomics/data!

**Comment by mstuart1** [comment]
I'm about to begin looking into these but as I started to make notes, I remembered that you had looser criteria for your samples than I did, so samples that were filtered out of my data set were included in yours and that might be why these samples don't have gen_ids.  

When we discussed creating the gen_id column, Malin said to use the stricter criteria to consider a sample successfully sequenced.  

This is also why you have a couple thousand loci in your past analysis compared to the ~1000 (or 860?) that I filtered out at that time.
